CHANGELOG.md

```diff
@@ -180,6 +177,9 @@ Adding a new version? You'll need three changes:
 - Support `ExtensionRef` HTTPRoute filter. It is now possibile to set a KongPlugin
   reference in the `HTTPRoute`s' `ExtensionRef` filter field.
   [#4838](https://github.com/Kong/kubernetes-ingress-controller/pull/4838)
+- Added `--kong-admin-token-file` flag to provide the Kong admin token via file
+  [Providing Kong admin token via file](https://github.com/Kong/deck/blob/main/CHANGELOG.md#v1120).
```

Comment:
Wait, why does that reference the deck changelog? I cannot see any relevant entry there 🤔

Reply:
No idea either; I hadn't looked at the linked content earlier. I agree there's nothing obviously relevant there.